### ISSUE TITLE
FAQ enchancement

### ISSUE THREAD

**Issue by mbettan** [CloudSlang/CloudSlang.github.io#174]
Hello,

It would be great to add a FAQ enchancement to answers basic questions on the CloudSlang like 

What is CloudSlang ?
What are the use cases and focus for CloudSlang?
How is CloudSlang different than configuration management tools like Chef, Puppet, SaltStack, and Ansible? Can I use it with these tools?
How is CloudSlang different from PaaS platforms like Cloud Foundry or OpenShift?
Is CloudSlang provided as a service?
What’s the quickest way to try CloudSlang out?
What is the difference between Orchestration and Configuration Management ?
What is the difference between Orchestration for containers like Kubernetes or Mesos and orchestrating IT Operations with CloudSlang ? 
What are the prerequisites for CloudSlang ? How can I install it ?
Which langages are supported by CloudSlang?
What clouds does CloudSlang support anyway? 
Where can I get help for CloudSlang ?
What other technologies does CloudSlang integrate with?
What is HPE Operations Orchestration and CloudSlang relationship ?
Why did you release HPE Operations Orchestration engine to Open source ?
How about support?
Anything else I should know?
Can you explain how did you ensure content testing and quality ?

**Comment by SamMarkowitz** [comment]
I agree. This is a good idea.

**Comment by SamMarkowitz** [comment]
I added a branch called **faq** to the docs repository (we can always add a link from the site) with a new faq page added to the docs and the list of questions we currently want to address. (I also changed the order a bit to group similar questions together.)

I will start filling in answers, but of course @mbettan you're welcome to contribute your own answers, especially if you have a specific idea of what the answer should look like.

Also @gfachler, you might want to take a look at the list and see if there any questions you might want to add/remove of bump up/down.
